test/mobile/custom_build/build.sh

```diff
@@ -54,6 +57,22 @@ run_default_build() {
     "${SRC_ROOT}/scripts/build_mobile.sh"
 }
 
+run_custom_build_with_static_dispatch() {
+  LIBTORCH_BUILD_ROOT="${BUILD_ROOT}/build_custom_libtorch_static"
+  LIBTORCH_INSTALL_PREFIX="${LIBTORCH_BUILD_ROOT}/install"
+
+  # Here we omitted the OP_DEPENDENCY flag so it generates registration
+  # code for used ROOT ops only, whose unboxing kernels are still needed
+  # by the JIT runtime. The intermediate ops will be automatically kepted
+  # by the linker as they are statically referenced by the static dispatch
+  # code, for which we can bypass the registration.
+  BUILD_ROOT="${LIBTORCH_BUILD_ROOT}" \
+    "${SRC_ROOT}/scripts/build_mobile.sh" \
+    -DCMAKE_CXX_FLAGS="-DSTRIP_ERROR_MESSAGES" \
+    -DSTATIC_DISPATCH_BACKENDS=CPU \
```

Comment:
Do you think that testing CPU static dispatch in CI is representative enough that we don't need to bother separately testing static dispatch for other backends? Maybe it is- if not though, we might want to either add tests for other backends, or enforce somewhere (e.g. codegen) that only CPU is allowed for now, until a use case for cuda static dispatch shows up somewhere.

Reply:
That's an idea - gate the codegen so it's only available (and tested) on the backends that current customers need. When somebody asks for another backend, we enable it and add tests which use that customer's build setup :)